src/org/rascalmpl/library/lang/java/m3/Core.rsc

```diff
@@ -117,7 +117,7 @@ M3 createM3FromDirectory(loc project, bool errorRecovery = false, bool includeJa
     registerProject(project, result);
     
     if (includeJarModels) {
-      results = composeJavaM3(project, {result, *{createM3FromJar(j, classPaths) |  j <- classPaths}});
+      results = composeJavaM3(project, {result, *{createM3FromJar(j, classPath=classPaths) |  j <- classPaths}});
```

Comment:
so, this is not related to this branch right? just accidental changes?

Reply:
Yes. My bad. Saw it and couldn't leave it there to rot but also didn't have the energy to checkout another branch.

Reply:
okay, we've all been there.